Sources/CombineSchedulers/TestScheduler.swift

```diff
@@ -257,6 +257,12 @@
       // NB: `DispatchTime(uptimeNanoseconds: 0) == .now())`. Use `1` for consistency.
       .init(now: .init(.init(uptimeNanoseconds: 1)))
     }
+
+    /// A test scheduler compatible with type erased UI schedulers.
+    public static var uiTest: TestScheduler<DispatchQueue.SchedulerTimeType, Never> {
+      // NB: `DispatchTime(uptimeNanoseconds: 0) == .now())`. Use `1` for consistency.
+      .init(now: .init(.init(uptimeNanoseconds: 1)))
+    }
```

Comment:
And instead of `DispatchQueue.uiTest`, how about `UIScheduler.test`?

```suggestion
  }
  
  extension UIScheduler {
    /// A test scheduler compatible with type erased UI schedulers.
    public static var test: TestScheduler<DispatchQueue.SchedulerTimeType, Never> {
      // NB: `DispatchTime(uptimeNanoseconds: 0) == .now())`. Use `1` for consistency.
      .init(now: .init(.init(uptimeNanoseconds: 1)))
    }
```